packages/framework/src/Model/Product/Search/ProductFilterCountDataElasticsearchRepository.php

```diff
@@ -158,7 +158,7 @@ protected function calculateFlagsPlusNumbers(ProductFilterData $productFilterDat
             $flagIds[] = $flag->getId();
         }
         $flagsPlusNumberResult = $this->client->search($plusFlagsQuery->getFlagsPlusNumbersQuery($flagIds));
-        return $this->aggregationResultToCountDataTransformer->translateFlagsPlusNumbers($flagsPlusNumberResult);
```

Comment:
I think we could add empty line above returns as part of this change. :slightly_smiling_face:

Reply:
I have added empty lines :)